build.proj

```diff
@@ -26,21 +26,27 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.cs" />
-    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.json" />
+    <DotnetCliBuildFrameworkInputs Include="build_projects/**/*.cs" Exclude="build_projects/**/obj/**/*.cs" />
+    <DotnetCliBuildFrameworkInputs Include="build_projects/**/project.json;build_projects/**/project.lock.json" />
   </ItemGroup>
+  
+  <!-- Workaround to "Native image cannot be loaded multiple times" issue 
+       A target in the top level file needs to run and invoke a task -->
+  <Target Name="MSBuildWorkaroundTarget">
+    <Message Text="Dont remove this target" />
+  </Target>
```

Comment:
Let me file a GH issue. Has previously been reported through contact with the team.